packages/sources/finage-test/src/endpoint/forex.ts

```diff
@@ -4,6 +4,35 @@ import overrides from '../config/overrides.json'
 import { httpTransport } from '../transport/forex-http'
 import { ForexBaseEndpointTypes, forexPriceInputParameters } from './utils'
 import { wsTransport } from '../transport/forex-ws'
+import { AdapterRequest } from '@chainlink/external-adapter-framework/util'
+import { PriceAdapterRequest } from '../index'
+
+/* 
+excludesMap includes currency pairs {quote: [base ...]} which should not be inverted.
+*/
+const excludesMap: Record<string, string[]> = {
+  USD: ['EUR', 'GBP', 'AUD', 'NZD', 'XAG', 'XAU', 'XPT', 'XPD', 'XCU'],
+}
+
+export const forexReqTransformer = (req: AdapterRequest<any>): void => {
+  if (req.requestContext.endpointName == 'forex') {
```

Comment:
Minor: Can we reference the name directly, to ensure one string doesn't change without the other?

```suggestion
  if (req.requestContext.endpointName == endpoint.name) {
```